commit b4c97232a5625b340c39d5cbf3157dde62d50508
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-04-02T05:22:40+00:00 (6 months ago)
    
    TRITON-1363 logarchiver config-agent should look for Hermes at /opt/smartdc/hermes, not /opt/triton/hermes
    Reviewed by: Dylan Yep <dyep49@gmail.com>
    Approved by: Dylan Yep <dyep49@gmail.com>

diff --git a/Makefile b/Makefile
index 8f250a0..39489d6 100644
--- a/Makefile
+++ b/Makefile
@@ -79,21 +79,19 @@ hermes:
 .PHONY: release
 release: all hermes
 	@echo "Building $(RELEASE_TARBALL)"
-	@mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)
 	@mkdir -p $(RELSTAGEDIR)/site
 	@touch $(RELSTAGEDIR)/site/.do-not-delete-me
-	cp -r $(ROOT)/node_modules \
-		$(ROOT)/package.json \
-		$(ROOT)/build \
-		$(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/
-	mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/boot
+	@mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/boot
 	cp -R $(ROOT)/deps/sdc-scripts/* $(RELSTAGEDIR)/root/opt/smartdc/boot
 	cp -R $(ROOT)/boot/* $(RELSTAGEDIR)/root/opt/smartdc/boot
-	cp -r $(TOP)/build/hermes/opt/smartdc/hermes \
+	cp -R $(TOP)/build/hermes/opt/smartdc/hermes \
 		$(RELSTAGEDIR)/root/opt/smartdc/hermes
-	mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/hermes/etc
+	@mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/hermes/etc
 	cp $(TOP)/etc/logsets.json \
 		$(RELSTAGEDIR)/root/opt/smartdc/hermes/etc
+	cp -R $(ROOT)/build \
+		$(RELSTAGEDIR)/root/opt/smartdc/hermes/
+	@rm -rf $(RELSTAGEDIR)/root/opt/smartdc/hermes/build/hermes
 	(cd $(RELSTAGEDIR) && $(TAR) -I pigz -cf $(ROOT)/$(RELEASE_TARBALL) root site)
 	@rm -rf $(RELSTAGEDIR)
 
diff --git a/boot/setup.sh b/boot/setup.sh
index c06fe57..9ba9be1 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -18,7 +18,7 @@ if [[ -h $SOURCE ]]; then
     SOURCE="$(readlink "$SOURCE")"
 fi
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-SVC_ROOT=/opt/triton/logarchiver
+SVC_ROOT=/opt/smartdc/hermes
 
 LOGARCHIVER_CFG=$SVC_ROOT/etc/config.json
 ZONE_UUID=`/usr/bin/zonename`
@@ -32,8 +32,8 @@ source /opt/smartdc/boot/lib/util.sh
 sdc_common_setup
 
 # set up services
-svccfg import /opt/smartdc/hermes/smf/hermes.xml
-svccfg import /opt/smartdc/hermes/smf/hermes-proxy.xml
+svccfg import $SVC_ROOT/smf/hermes.xml
+svccfg import $SVC_ROOT/smf/hermes-proxy.xml
 
 # add log rotation entries for services
 sdc_log_rotation_add amon-agent /var/svc/log/*amon-agent*.log 1g
